### DISCUSSION TITLE
General Chat About The Next JS and Strapi Starter

### DISCUSSION THREAD

**Discussion by PaulBratslavsky** [strapi/nextjs-corporate-starter#35] [General]
General discussion about the project.

**Comment by PaulBratslavsky** [comment]
Wanted to share here just in case you were wondering.  When populating our home page, we are not handling it from the frontend and instead we are relying on a route middleware in Strapi.  You can find an example [here](https://github.com/strapi/nextjs-corporate-starter/blob/main/backend/src/api/page/middlewares/page-populate-middleware.js)

If you like to learn more check out this article by Kellen [here](https://strapi.io/blog/route-based-middleware-to-handle-default-population-query-logic)

**Comment by MrCaro** [comment]
Do you guys have any recommendations and share `how to` for hosting this Strapi App Starter outside of Strapi Cloud? I checked out `Cleavr`  and seems you can connect it to your `DO` droplet but I'm having issues when deploying. A quick `how to host this app` would be terrific!

thanks!

**Comment by roladev** [comment]
How can I run this project in Postgres?

It's a simple way to remove /en/ ([lang]) url path?

**Reply by PaulBratslavsky** [reply]
You can take a look at this example that shows the configuration for PostreSQL.
https://docs.strapi.io/dev-docs/configurations/database#configuration-examples

but I will make time and record the video showcasing the process.

**Reply by PaulBratslavsky** [reply]
@RolandaDev here is the [video](https://youtu.be/ROvjBdI8-MM?feature=shared) hopes it helps.

**Reply by roladev** [reply]
Aww!! You are extremely kind. Thanks

**Reply by PaulBratslavsky** [reply]
You are welcome. It was a great question. And will benefit others wondering the same thing.  I am going to post this video this week on our youtube channel.

**Comment by Jullew** [comment]
Is there a guide how to deploy both backend and frontend to different solution than Strapi Cloud?

**Reply by PaulBratslavsky** [reply]
Yes, you can easily deploy Strapi to Digital Ocean, Render or other similar solutions.
Strapi to Digital Ocean Droplet: https://docs.strapi.io/dev-docs/deployment/digitalocean
Strapi to Digital Ocean App platform: https://docs.strapi.io/dev-docs/deployment/digitalocean-app-platform
Render: https://docs.render.com/deploy-strapi

@Jullew hope this helps.

**Comment by itsdagi** [comment]
How can i change the strapi logo on the page

**Reply by PaulBratslavsky** [reply]
If you are referring on the front end top nav and footer you should be able to change it with in Strapi Admin inside the Global single collection type, that is where that data is stored.

**Reply by itsdagi** [reply]
> If you are referring on the front end top nav and footer you should be able to change it with in Strapi Admin inside the Global single collection type, that is where that data is stored.

Thank u

**Comment by aayukarki** [comment]
why is elements not working when we create a new one? is there something that needs to be done after creating elements. the data is just not showing.

**Comment by PaulBratslavsky** [comment]
@aayukarki can you provide an example of what you are trying to create.

**Reply by aayukarki** [reply]
Hi Paul,

![image](https://github.com/strapi/nextjs-corporate-starter/assets/99771695/6a6ba127-b104-4821-a92c-b27438d49ec3)
I am trying to create new elements. and the data is not showing up 

![image](https://github.com/strapi/nextjs-corporate-starter/assets/99771695/890c16e2-2d26-4e66-af1a-8fad479e3205)
title is showing on frontend but
award values are not displaying at all on the frontend.

![image](https://github.com/strapi/nextjs-corporate-starter/assets/99771695/2187c69f-847e-4939-ab0d-f43c18de5403)

**Reply by PaulBratslavsky** [reply]
Can you show me the code for fetching this data? You must update the query to populate the "award" component. Is this being displayed on the home page or blog?

**Comment by webmaffia** [comment]
Hi,

Pages load slowly the first time, but after that they load smoothly.

Please provide a solution on how I can load a page quickly.

And how can I remove the "en" from the path?
